pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -876,3 +879,104 @@ func (c updateRulesOrderInGroup) Exec(sess *xorm.Session, migrator *migrator.Mig
 	}
 	return nil
 }
+
+func AddExternalAlertmanagerToDatasourceMigration(mg *migrator.Migrator) {
+	mg.AddMigration("migrate external alertmanagers to datsourcse", &externalAlertmanagerToDatasources{})
+}
+
+type externalAlertmanagerToDatasources struct {
+	migrator.MigrationBase
+}
+
+// Just copy old struct code to use Alertmanagers as new one is not getting it.
```

Comment:
What does this mean? 😄

Reply:
I needed to copy the old AdminConficuration struct as the new one has not the attribute `Alertmanagers []string` defined in it anymore.